torch/testing/_comparison.py

```diff
@@ -990,7 +1005,6 @@ def _compare_regular_values_close(
         identifier: Optional[Union[str, Callable[[str], str]]] = None,
     ) -> None:
         """Checks if the values of two tensors are close up to a desired tolerance."""
-        actual, expected = self._promote_for_comparison(actual, expected)
```

Comment:
We unconditionally upcasted here in the past, since that was needed for `isclose`. This is no longer the case and so we can just drop that.